src/api/Metadata.js

```diff
@@ -257,6 +267,71 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * Gets classification for a file.
+     *
+     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @param {boolean} forceFetch - True to clear the cache and make an api call
+     * @param {boolean} refreshCache - True to
+     * @return {Promise}
+     */
+    async getClassification(
+        file: BoxItem,
+        successCallback: Function,
+        errorCallback: Function,
+        forceFetch: boolean = false,
```

Comment:
{ forceFetch, refreshCache } as object options. See File.js
flowtype is FetchOptions